<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ATSYSTEMS-PC" ignored="0" name="Test" tests="6" failures="2" timestamp="14 Feb 2022 07:23:23 GMT" time="0.1" errors="0">
  <testcase name="Test01" time="0.018" classname="selbatch2.DemoTest"/>
  <testcase name="Test02" time="0.005" classname="selbatch2.DemoTest">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]at selbatch2.DemoTest.Test02(DemoTest.java:19)... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test02 -->
  <testcase name="Test03" time="0.005" classname="selbatch2.DemoTest">
    <skipped/>
  </testcase> <!-- Test03 -->
  <testcase name="Test01" time="0.002" classname="selbatch2.DemoTest2"/>
  <testcase name="Test02" time="0.003" classname="selbatch2.DemoTest2">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]at selbatch2.DemoTest2.Test02(DemoTest2.java:20)... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test02 -->
  <testcase name="Test03" time="0.003" classname="selbatch2.DemoTest2">
    <skipped/>
  </testcase> <!-- Test03 -->
</testsuite> <!-- Test -->
